utils/src/wizard/InputRevalidationStep.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardPromptStep, IActionContext } from "../..";
+import { localize } from "../localize";
+
+export class InputRevalidationStep<T extends IActionContext> extends AzureWizardPromptStep<T> {
+    constructor(private readonly _key: string, private readonly _isPassword?: boolean) {
+        super();
+    }
+
+    public async prompt(context: T): Promise<void> {
+        await context.ui.showInputBox({
+            prompt: localize('verifyPreviousInput', 'Please confirm by re-entering the previous value.'),
```

Comment:
Should the prompt value be customizable? I could see us wanting to make it say "Re-enter password to confirm" or something.